src/components/input/InputElement.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+
+
+const InputElement = ({ label, type, id, placeholder, description }) => {
+    const [inputValue, setInputValue] = useState('');
+    console.log(inputValue);
+
+    return (
+        <>
+            {
+                label &&
+                <label htmlFor={id}>{label} <br /></label>
```

Comment:
Remember it's going to have a property to handle the position of label (left or top)